lib/resourceapply/cv.go

```diff
@@ -62,7 +36,7 @@ func ApplyClusterVersionFromCache(ctx context.Context, lister configlistersv1.Cl
 		return existing, false, nil
 	}
 
-	klog.V(2).Infof("Updating ClusterVersion %s due to diff: %v", required.Name, diff.ObjectDiff(existing, required))
+	klog.V(2).Infof("Updating ClusterVersion %s due to diff: %v", required.Name, cmp.Diff(existing, required))
```

Comment:
no mode guard here?

Reply:
Unlike the other resources, the CV is created/updated from multiple controllers and the mode is not present there. So it cannot be used.